Exercises/JS/06-js-logical-operators/js/script.js

```diff
@@ -0,0 +1,11 @@
+let hour = Number(prompt ("Enter the current hour (out of 0-23):"));
```

Comment:
Good that you converted to "number" type before comparing.

Reply:
Thanks.